src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -116,6 +116,16 @@ export default class ExportToolbarAction extends AbstractListToolbarAction {
     };
 
     @action handleConfirm = () => {
+        let filters = {};
+        if (Object.keys(this.listStore.filterQueryOption).length > 0) {
+            filters = this.listStore.filterQueryOption;
+        }
+
+        let search = null;
+        if (this.listStore.searchTerm.get()) {
+            search = this.listStore.searchTerm.get();
+        }
```

Comment:
Since `get()` should return `undefined` already if nothing is passed, something like that should suffice:

```javascript
const search = this.listStore.searchTerm.get();
```

Reply:
Thanks for the tip @danrot! Refactored in latest commit.